src/DOM/File/FileList.purs

```diff
@@ -0,0 +1,25 @@
+module DOM.File.FileList
+  ( length
+  , item
+  , toArray
+  ) where
+
+import Prelude ((-))
+import Data.Array as Array
+import Data.Maybe (Maybe())
+import Data.Nullable (Nullable(), toMaybe)
+import DOM.File.Types (FileList(), File())
+
+-- | Number of files in the `FileList` object.
+foreign import length :: FileList -> Int
+
+-- | Get `File` at the certain position
+foreign import itemImpl :: Int -> FileList -> Nullable File
+
+item :: Int -> FileList -> Maybe File
+item i l = toMaybe (itemImpl i l)
+
+toArray :: FileList -> Array File
+toArray fileList = Array.mapMaybe
```

Comment:
I'm not sure whether we should include this, I think it's the only function in the entire library that is not a type coercion/foreign read/direct mapping to an API method.